routing_common/transit_serdes.hpp

```diff
@@ -131,8 +136,12 @@ class Deserializer
     t.Visit(*this);
   }
 
+  /// \note This method should be called beforw deserializing every table.
+  void ResetCache() { m_lastDecodedPoint = m2::PointD(); }
```

Comment:
s/ResetCache/Reset/

Reply:
Done

Reply:
This method is removed. Nothing to reset at Serializer/Deserializer now.